designs/inter-ident-verify.md

```diff
@@ -0,0 +1,98 @@
+# Interpersonal Identity Verification
```

Comment:
Do you need some help with rebasing/cherry-pick to clean up your branch?

Reply:
I think I know how to do it now. How should I go about removing it from this PR though?